flatgfa/tests/test.txt

```diff
@@ -0,0 +1 @@
+[A, C, T, G, G]
```

Comment:
This is just a note to myself: after landing this PR, maybe it would make sense to have the CLI output the raw nucleotides instead of a comma-separated list. Then the test could be a true round-trip, using the input file as the output file.